### PR TITLE
feat: update organization schematics

### LINKED ISSUE #9422: Update organization schematics
After https://github.com/SAP/spartacus/issues/9238, https://github.com/SAP/spartacus/issues/9421 and https://github.com/SAP/spartacus/pull/8018 are done, we should update the organization schematics to do the following:

- by default, the administrationmodule should be eagerly imported (import the root module)
- add an option to lazily import the administrationmodule (`ng add @spartacus/organization --lazy`)

### PR DESCRIPTION
**Pull request by znikola**
closes #9422

### CODE REVIEW

File: projects/schematics/src/shared/constants.ts [hunk 1]
```diff
@@ -56,7 +57,8 @@ export const COMPONENT_FACTORY_RESOLVER = 'ComponentFactoryResolver';
 export const HTTP_CLIENT = 'HttpClient';
 
 export const B2C_STOREFRONT_MODULE = 'B2cStorefrontModule';
-export const ORGANIZATION_MODULE = 'OrganizationModule';
+export const ADMINISTRATION_MODULE = 'AdministrationModule';
```
Comment (marlass) [inline projects/schematics/src/shared/constants.ts:60]: Why not keep these things in the organization schematics?

File: feature-libs/organization/schematics/add-organization/index_spec.ts [hunk 2]
```diff
@@ -56,53 +61,91 @@ describe('Spartacus Organization schematics: ng-add', () => {
...
+          throw new Error('angular.json missing?');
+        }
+
+        const angularJson = JSON.parse(buffer.toString(UTF_8));
+        const buildStyles: string[] =
+          angularJson.projects['schematics-test'].architect.build.options
+            .styles;
+        expect(buildStyles).toEqual([
+          'src/styles.scss',
+          'src/styles/organization.scss',
```
Comment (marlass) [inline feature-libs/organization/schematics/add-organization/index_spec.ts]: Shouldn't it be `spartacus-organization.scss`?

**Review by marlass — CHANGES_REQUESTED** [review]

File: projects/schematics/src/shared/constants.ts (hunk 1 above)
Comment (znikola) [reply projects/schematics/src/shared/constants.ts:60]: The idea is to make it re-usable for the migration in the future.

Comment (marlass) [reply projects/schematics/src/shared/constants.ts:60]: I don't understand. Migrations for the organization lib should be handled in the future in the organization lib.

Comment (znikola) [reply projects/schematics/src/shared/constants.ts:60]: Maybe, we haven't decided it yet.

**Review by marlass — APPROVED** [review]
Tested both lazy and eager version.
In those cases desired changes were made in application ✅ 

Only nitpick is to move organization related variables to @spartacus/organization library, but that can be done in the next PR.

**Pull request merged**